t/06elements-port.t

```diff
@@ -0,0 +1,555 @@
+# $Id$
+
+##
+# this test checks the DOM element and attribute interface of XML::LibXML
+
+use v6.c;
+use Test;
+
+#use base qw(ParentClass);
+#
+#sub new {
+#    my $class = shift;
+#    $class = ref $class if ref $class;
+#    my $self = bless {}, $class;
+#    $self;
+#}
+
+#1;
+
+# Should be 187.
+#use Test::More tests => 191;
+plan 187;
+
+
+use XML::LibXML;
+
+my $foo       = "foo";
+my $bar       = "bar";
+my $nsURI     = "http://foo";
+my $prefix    = "x";
+my $attname1  = "A";
+my $attvalue1 = "a";
+my $attname2  = "B";
+my $attvalue2 = "b";
+my $attname3  = "C";
+
+# TEST:$badnames=4;
+my @badnames= ("1A", "<><", "&", "-:");
+
+# 1. bound node
+my $doc = XML::LibXML::Document.new();
+my $elem = $doc.createElement( $foo );
+# TEST
```

Comment:
Vestigial comments from the original code. They serve no purpose, that I can tell.
I will gladly excise them.